### DISCUSSION TITLE
Flashlight mod size change showing odd values

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#16750] [Q&A] [answered]
![osu_2022-02-02_19-49-28](https://user-images.githubusercontent.com/74463310/152220416-09f10f51-53f8-4424-ba9f-953e60f87f6d.jpg)
as you can see flashlight seems to be somehow slightly increasing the value it is given erroneously, the reason i didnt pick up on this when coding it is because it doesnt happen in the mod select, only on the loading screen and in the plays list

**Comment by bdach** [comment]
fwiw i'm fixing this.

nothing is "increasing the value erroneously", it's just printed wrong. google how floating point math works with regard to decimals, i have explained it too many times already to do so again.

**Reply by mk56-spn** [reply]
i suspected this might be the case, my bad for going with the wrong explanation, and dw i wasnt expecting an explanation, again sorry for the bother

**Comment by bdach** [comment, marked as answer]
will be fixed via #16751
